torch/csrc/jit/interpreter.cpp

```diff
@@ -1044,23 +1044,37 @@ struct InterpreterStateImpl : c10::intrusive_ptr_target {
   }
 
   void formatStackTrace(std::ostream& out) {
-    for (size_t i = 0; i < frames.size(); ++i) {
+    std::string previous_fn_name = "";
+    for (int i = frames.size() - 1; i >= 0; i--) {
       const Frame& frame = frames[frames.size() - 1 - i];
       size_t pc = (i == 0) ? frame.pc
                            : frame.pc -
               1; // make sure we report the call node, not the node after it
       Node* node = frame.function->instructions_source_[pc];
-      if (i > 0) {
-        out << "during call ";
+      if (node->callstack()) {
+        for (const auto& p : (*node->callstack())->vec()) {
+          p.second.print_with_context(
```

Comment:
I know that Python doesn't add more context by default, but I think we should do it anyway. For instance, python notebooks will print more of the context in stack traces. My previous comments were mostly about the wording differences for the stuff around the highlighting. I also think we should keep the highlighting on, since it makes it easier to read which part of the line we are talking about.